assets/js/base/components/product-grid/container.js

```diff
@@ -25,27 +25,26 @@ class ProductGridContainer extends Component {
 	};
 
 	onOrderChange = ( event ) => {
-		const orderValue = event.target.value;
+		const sortValue = event.target.value;
 		this.setState( {
 			currentPage: 1,
-			orderValue,
+			sortValue,
 		} );
-		this.props.onOrderChange( orderValue );
+		this.props.onOrderChange( sortValue );
```

Comment:
should `onOrderChange` be modified to `onSortChange`?

Reply:
Good catch! Fixed in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1012/commits/03e85fcd889c30f58f18ec5b99f5e8c7027fe0dc.